### ISSUE TITLE
Thymeleaf documentation - not found github

### ISSUE THREAD

**Issue by cjcalmeida** [thymeleaf/thymeleaf-spring#250]
When you try to open thymeleaf site (https://thymeleaf.org), status code not found (404) is returned.

**Comment by ultraq** [comment]
Yup, looks like the website is currently down.  Trying to bring it up now...
